app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="ui small menu inverted">
+  <div class="ui container">
+    <%= link_to "Chatroom", root_path, class: "item" %>
+    <a class="item" alt="#">
+      Messages
+    </a>
+    <div class="right menu">
+      <div class="ui dropdown item">
+        Account <i class="dropdown icon"></i>
```

Comment:
personally, I hate it when the structure is like this.
Why not make it:
```
<span>Account</span>
<i class="dropdown icon"></i>
```
?
It'll be easier to target it in js, CSS and even semantically.